src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4180,6 +4218,15 @@ size_t os::large_page_size() {
   return _large_page_size;
 }
 
+size_t os::Linux::select_large_page_size(size_t bytes) {
```

Comment:
As mentioned by @tstuefe , this is duplicating `size_t os::page_size_for_region(size_t region_size, size_t min_pages, bool must_be_aligned) `

Reply:
In latest patch I removed os::Linux::select_large_page_size and use os::page_size_for_region instead.